packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -360,6 +341,9 @@ export const getStyles = memoizeFunction(
             '&.is-open': {
               borderColor: root.borderColor,
               selectors: {
+                ':after': {

```

Comment:
@betrue-final-final and @jspurlin  just to verify: when the combobox is open, should focus rect be around the input or not? Hypothetically the input still has focus (you can type there), though if we follow Dropdown's example, there is no focus border when it is open.

If we ARE removing the focus border when it is open, it seems the chevron button has a bit of a gap on the top:

![image](https://user-images.githubusercontent.com/1434956/71851939-7d105b00-308c-11ea-8be1-fb854ebce687.png)

Reply:
good catch on the style of the chevron button. once i get confirmation from design, i will address accordingly

Reply:
The focus rect should still be around the whole combobox since "it" (e.g. the input) has focus

Reply:
Ok, that makes sense, sort of. Combo is always a little confusing. I assume that means that Dropdown remains as it is today without focus as the menu has focus?

Reply:
hmm. In dropdown case, menu has the focus. In combobox case, the entire combobox has focus (both the input and the menu has focus).

Reply:
Sounds good.

Reply:
thinking a bit more and checked with Ben, will add focus border (remove the redundant black border) back for ComboBox, dropdown remain the same

Reply:
Per @maxwellred's earlier comment https://github.com/OfficeDev/office-ui-fabric-react/pull/11497#issuecomment-567603193, the current design in the toolkits is that ComboBox should NOT have the focus border when open.

Reply:
Latest: 
![image](https://user-images.githubusercontent.com/1207059/71854309-8f8d9300-3092-11ea-959d-e06fdc5e2904.png)

Reply:
@micahgodbolt - on the chevron button style issue you pointed out, it's unrelated to this PR so im going to issue a separate PR to fix instead of adding more changes to this one

Reply:
The toolkit should be updated @maxwellred